host/3.0/dotnet-build.yml

```diff
@@ -63,13 +63,50 @@ steps:
       IMAGE_NAME=azurefunctions.azurecr.io/azure-functions/3.0/dotnet:$(Build.SourceBranchName)-appservice
 
       docker build -t $IMAGE_NAME \
-                  -f host/3.0/buster/amd64/dotnet/dotnet-appservice.Dockerfile \
-                  host/3.0/buster/amd64/dotnet/
+                  -f host/3.0/buster/amd64/dotnet/dotnet-inproc/dotnet-appservice.Dockerfile \
+                  host/3.0/buster/amd64/dotnet/dotnet-inproc/
       npm run test $IMAGE_NAME --prefix  test/
       docker push $IMAGE_NAME
     displayName: dotnet-appservice
     continueOnError: false
 
+
```

Comment:
I could break this out and create a new DevOps pipeline and `dotnet-isolated-build.yml`.

Reply:
up to you :)

Reply:
Let's leave this here to avoid creating a new pipeline. And if we need to later, should be easily moveable.